tests/xharness/AppRunner.cs

```diff
@@ -423,15 +426,16 @@ public bool TestsSucceeded (AppInformation appInfo, string test_log_path, bool t
 
 		public async Task<int> RunAsync ()
 		{
-			CrashReportSnapshot crash_reports;
 			ILog device_system_log = null;
 			ILog listener_log = null;
 			ILog run_log = MainLog;
 
 			if (!isSimulator)
 				FindDevice ();
 
-			crash_reports = new CrashReportSnapshot (harness, MainLog, Logs, isDevice: !isSimulator, deviceName);
+			var crashLogs = new Logs (Logs.Directory);
+
+			ICrashSnapshotReporter crash_reports = snapshotReporterFactory.Create (MainLog, crashLogs, isDevice: !isSimulator, deviceName);
```

Comment:
Did you confirm that crash reports show up in the html report now? It looks like they're just added into a separate Logs instance, which no other code knows about.

Reply:
You're right. I added a fix and will verify locally